packages/url/urltestgenerator.html

```diff
@@ -26,11 +26,11 @@
         continue // only true for "_password"
       }
       if(output !== "")
-        output += " "
+        {output += " "}
       output += componentsKeys[i] + ":" + esc(val)
     } else if(components[i] == "_password") {
```

Comment:
Is prettier set up to run on this file? This isn't something I'd expect it to produce.

Reply:
Yes! In all files: https://github.com/webcomponents/polyfills/pull/405/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R13